tests/conftest.py

```diff
@@ -95,6 +95,10 @@ def gaussian_device(n_subsystems):
     """Number of qubits or modes."""
     return DummyDevice(wires=n_subsystems)
 
+@pytest.fixture(scope="session")
+def gaussian_dummy():
+    """Number of qubits or modes."""
+    return DummyDevice
```

Comment:
How come the above fixture couldn't be used?

Reply:
Because I need to set the number of subsystems in the test!

Reply:
Ah, you should be able to do it with the other one. It takes a `num_subsystems` fixture, so you just need to tell the test the value(s), i.e., `@pytest.mark.parametrize(num_subsystems, [3])`.

Assuming the number of subsystems doesn't change dynamically within the test, in which case, the way you have it is probably the clearest way.